influxql/iterator.gen.go

```diff
@@ -767,7 +767,7 @@ func (itr *floatInterruptIterator) Next() (*FloatPoint, error) {
 	if itr.count&0xFF == 0xFF {
 		select {
 		case <-itr.closing:
-			return nil, nil
+			return nil, itr.Close()
```

Comment:
Ok, I don't think there are any iterators that can't accept a double close. If there are, we can add a check to the interrupt iterator so it will only perform a single close.